### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by sethtara**
added few more languages.

### CODE REVIEW

**Comment by sethtara** [comment]
will be adding more languages soon .

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    <string name="pref_storm_alert_sensitivity_title">حساسية تنبيه العاصفة</string>
+    <string name="pref_show_ruler" translatable="false">pref_show_ruler</string>
+    <string name="pref_show_ruler_title">إظهار المسطرة</string>
+    <string name="pref_show_sun_moon_compass" translatable="false">pref_show_sun_moon_compass</string>
+    <string name="pref_show_sun_moon_compass_title">عرض الشمس /القمر على البوصلة</string>
+    <string name="pref_show_linear_compass" translatable="false">pref_show_linear_compass</string>
+    <string name="pref_show_linear_compass_title">إظهار البوصلة الخطية</string>
+    <string name="pref_show_linear_compass_summary">عندما يكون الهاتف عموديًا</string>
+
+    <string name="delete_beacon_confirmation">هل أنت متأكد أنك تريد إزالة "٪s"؟</string>
```
Comment (kylecorry31) [inline app/src/main/res/values-ar/strings.xml:94]: The %s is messed up, causing the build to fail

File: app/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    <string name="pref_storm_alert_sensitivity_title">风暴警报敏感性</string>
+    <string name="pref_show_ruler" translatable="false">pref_show_ruler</string>
+    <string name="pref_show_ruler_title">显示标尺</string>
+    <string name="pref_show_sun_moon_compass" translatable="false">pref_show_sun_moon_compass</string>
+    <string name="pref_show_sun_moon_compass_title">在指南针上显示太阳/月亮</string>
+    <string name="pref_show_linear_compass" translatable="false">pref_show_linear_compass</string>
+    <string name="pref_show_linear_compass_title">显示线性罗盘</string>
+    <string name="pref_show_linear_compass_summary">当手机垂直时</string>
+
+    <string name="delete_beacon_confirmation">您确定要删除"％s"吗？</string>
```
Comment (kylecorry31) [inline app/src/main/res/values-zh/strings.xml:94]: The %s is messed up here, causing the build to fail

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    <plurals name="last_hours">
+        <item quantity="one">الاخير %d ساعة</item>
```
Comment (kylecorry31) [inline app/src/main/res/values-ar/strings.xml:97]: This needs to have the following quantities defined here: few, many, two, zero

**Comment by kylecorry31** [comment]
FYI, that failures being generated are from the gradle android lint comment which you can run via Android Studio (app > tasks > verification > lint)

File: app/src/main/res/values-ar/strings.xml (hunk 3 above)
Comment (sethtara) [reply app/src/main/res/values-ar/strings.xml:97]: there is all the quantities defined as defined in the main strings.xml

**Comment by sethtara** [comment]
### Finally All Checks have passed :joy:

**Pull request merged**